prod-aws/kafka-shared-msk/README.md

```diff
@@ -7,8 +7,14 @@ Use the [tls-app](../../modules/tls-app) module for granting access for the apps
 
 ## Contributing
 
-We want utilitywarehouse/pubsub to own any chages to the msk clusters so please don't make any CODEOWNERS changes as
-part PRs adding resouces to the clusters.
+We want `utilitywarehouse/pubsub` to own any changes to the MSK clusters so please don't make any CODEOWNERS changes as
+part PRs adding resources to the clusters.
+
+We have a custom `tflint` plugin that help enforce best practices and avoid some
+common issues. This is run through `pre-commit`, see [the contributing
+docs](../../CONTRIBUTING.md) for installation details. Each rule is documented
```

Comment:
The README contains an index with all the rules. I think it would be better to link it here.

Reply:
> The README contains an index with all the rules. I think it would be better to link it here.

[33d0756](https://github.com/utilitywarehouse/kafka-cluster-config/pull/635/commits/33d07560fcf6ff3ae2879c21e7eecee0b6b7cc4f)